test/docfx.Tests/BuildCommandTest.cs

```diff
@@ -119,5 +119,128 @@ public void TestGetConfigWithNoInputAndDocfxJsonExists()
                 },
                 File.ReadAllLines(file));
         }
+
+        [Fact]
+        [Trait("Related", "docfx")]
+        public void TestParseCommandOption()
+        {
+            var globalMetadataFile1 = Path.Combine(_outputFolder, "global1.json");
+            File.WriteAllLines(globalMetadataFile1, new string[]
+            {
+                "{",
+                "  \"key\": \"global1.json\",",
+                "  \"global1\": \"1\"",
+                "}",
+            });
+
+            var fileMetadataFile1 = Path.Combine(_outputFolder, "file1.json");
+            File.WriteAllLines(fileMetadataFile1, new string[]
+            {
+                "{",
+                "  \"key\": {",
+                "    \"filepattern1\": \"file1.json\"",
+                "  }",
+                "}",
+            });
+
+            var globalMetadataFile2 = Path.Combine(_outputFolder, "global2.json");
+            File.WriteAllLines(globalMetadataFile2, new string[]
+            {
+                "{",
+                "  \"key\": \"global2.json\",",
+                "  \"global2\": \"2\"",
+                "}",
+            });
+
+            var fileMetadataFile2 = Path.Combine(_outputFolder, "file2.json");
```

Comment:
no need, the files in folder generated from `GetRandomFolder` will be auto-destroyed

---
In reply to: [73813867](https://github.com/dotnet/docfx/pull/575#discussion_r73813867) [](ancestors = 73813867)